### ISSUE TITLE
a question about hdf5

### ISSUE THREAD

**Issue by yyytq** [RiweiChen/DeepFace#10]
I am having a very strange problem with hdf5.
it will return an "IOError: unable to create file (File accessibility: unable to open file)". Here is the offending line of code:
h5py.File('/path/to/file', 'w')
I think the filepath is right and i am not trying to create a file that already exists. do you have any idea about this problem?
